include/translatable-object.php

```diff
@@ -79,8 +97,10 @@ abstract class PLL_Translatable_Object {
 	 * @param PLL_Model $model Instance of `PLL_Model`, passed by reference.
 	 */
 	public function __construct( PLL_Model &$model ) {
-		$this->model          = $model;
-		$this->tax_to_cache[] = $this->tax_language;
+		$this->languages_model = $model->languages;
```

Comment:
```suggestion
		$this->languages = $model->languages;
```

Reply:
Done in [8b93699](https://github.com/polylang/polylang/pull/1503/commits/8b93699e5e88c947b96e9805938be8b631939b5d).